client_cert.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import "crypto/tls"
+
+var (
+	tlsLoadX509KeyPair = tls.LoadX509KeyPair
+)
+
+// readClientCert - helper function to read client certificate
+// from pem formatted certPath and keyPath files
+func readClientCert(certPath, keyPath string) ([]tls.Certificate, error) {
+	if certPath == "" || keyPath == "" {
```

Comment:
Actually, there are 3 scenarios:
* user specified none - we should return `nil, nil` in this case;
* user specified one of paths - we should print an error describing what happened. A good place to check this would be in `config#checkArgs`, I think. You can create named errors, like `errNoPathToKey` and `errNoPathToCert`(don't forget to write some tests:wink:);
* user specified both - proceed to `tls.LoadX509KeyPair`.